libraries/botbuilder-azure/src/azureBlobTranscriptStore.ts

```diff
@@ -143,6 +144,7 @@ export class AzureBlobTranscriptStore implements TranscriptStore {
      *
      * @param channelId Channel Id.
      * @param continuationToken ContinuationToken token to page through results.
+     * @returns A promise representation of [PagedResult<TranscriptInfo>](xref:botbuilder-core.PagedResult<T>)
```

Comment:
```suggestion
     * @returns A promise representation of [PagedResult<TranscriptInfo>](xref:botbuilder-core.PagedResult)
```

I think this should be like this. I found some samples in the adaptive-expressions library that doesn't use the <T> in the xref:
https://github.com/microsoft/botbuilder-js/blob/c76d08118f9c98ac5590767f677420c8f738feb5/libraries/adaptive-expressions/src/expressionProperties/arrayExpression.ts#L20

Reply:
Changed. Thanks!